lua/nvim-treesitter-refactor/navigation.lua

```diff
@@ -16,7 +16,7 @@ function M.goto_definition(bufnr, fallback_function)
 
   local definition = locals.find_definition(node_at_point, bufnr)
 
-  if fallback_function and definition.id == node_at_point.id then
+  if fallback_function and definition == node_at_point then
```

Comment:
Doesn't this require both checks? I think there can be different node object referring to the same node.

Reply:
Looks like the code here should have been `node:id()`, but from the docs it says

> The id is not guaranteed to be unique for nodes from different trees.

Don't know if the instance of the node would ever change here.